JS/modules/dateTime.js

```diff
@@ -0,0 +1,30 @@
+import { DateTime } from "./luxon.js";
+
+const dateLine = document.querySelector(".dayDate");
+const navBar = document.querySelector("nav");
+/* eslint-disable-next-line */
+export const displayDate = () => {
+  const dt = DateTime.now();
+  const f = { month: "long", day: "numeric" };
+  dt.setLocale("en-US").toLocaleString(f);
+  dateLine.innerHTML = `${dt}`;
```

Comment:
- [ ] I would like to suggest you to please format your `Date` and `Time` properly so that a normal user who uses your website can understand the date and time `properly`. It seems like you are just passing the **raw** date and time strings directly into your `HTML` which seems too difficult to understand for the non-technical person. Kindly format it in this format `DD-MM, YY HH:MM:SS` or you can choose other format but not the default `date and time` string.

### Problem
![image](https://github.com/GeekyHacks/AwesomeBooks_ES6/assets/66476329/6f1bff19-dc6e-460c-8581-ba5139555be7)